res_partner_attributes_add_NEQ/i18n/nb_NO.po

```diff
@@ -0,0 +1,34 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# * res_partner_attributes_add_NEQ
+# 
+# Translators:
+# Imre Kristoffer Eilertsen <imreeil42@gmail.com>, 2016
+msgid ""
+msgstr ""
+"Project-Id-Version: l10n-canada (8.0)\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2016-07-30 19:57+0000\n"
+"PO-Revision-Date: 2016-08-04 04:49+0000\n"
+"Last-Translator: Imre Kristoffer Eilertsen <imreeil42@gmail.com>\n"
+"Language-Team: Norwegian Bokmål (Norway) (http://www.transifex.com/oca/OCA-l10n-canada-8-0/language/nb_NO/)\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Language: nb_NO\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#. module: res_partner_attributes_add_NEQ
+#: field:res.partner,neq:0
+msgid "NEQ"
+msgstr "NEW"
+
+#. module: res_partner_attributes_add_NEQ
+#: model:ir.model,name:res_partner_attributes_add_NEQ.model_res_partner
+msgid "Partner"
+msgstr "Partner"
+
+#. module: res_partner_attributes_add_NEQ
+#: help:res.partner,neq:0
+msgid "Quebec Enterprise Number (10 digits)"
+msgstr "Québec-firmanummer (10 sifre)"
```

Comment:
@dufresnedavid Do you mean it makes no sense that this module has Norwegian translations?